src/network/meson.build

```diff
@@ -56,8 +56,9 @@ sources = files(
         'networkd-link.c',
         'networkd-lldp-rx.c',
         'networkd-lldp-tx.c',
-        'networkd-manager-bus.c',
         'networkd-manager.c',
+        'networkd-manager-bus.c',
```

Comment:
Not a matter, but here you sort `-` after `.`, but in below you use the inverse order.